hardware_interface/include/fake_components/generic_system.hpp

```diff
@@ -78,6 +78,15 @@ class GenericSystem : public hardware_interface::BaseInterface<hardware_interfac
     hardware_interface::HW_IF_ACCELERATION,
     hardware_interface::HW_IF_EFFORT
   };
+
+  struct MimicJoint
+  {
+    std::size_t joint_index{};
+    std::size_t mimicked_joint_index{};
```

Comment:
```suggestion
    std::size_t joint_index;
    std::size_t mimicked_joint_index;
```
please set default values with `=` and let's use `' '` to denote nothing (as opposed to `'{}'` :D )